### ISSUE TITLE
Save full signature file in the SBT leaves?

### ISSUE THREAD

**Issue by luizirber** [dib-lab/sourmash#198]
Right now, the SBT `save` command will only save the signature for the specific `k` it was built with. We might want to save all the signatures in the original file (multiple `k`, potentially), to be able to share leaves between trees.

Thinking this because I created trees for `refseq-k{21,31,51}`, and they could all share the same leaves if it was maintained as the original input file.

It is probably a good idea to expose this to `sbt_index`, there might be cases were it is better to just save the specific `k`.

**Comment by ctb** [comment]
Good idea, I think.  Do we want to minify the JS or otherwise do something
compressive with things?

**Comment by luizirber** [comment]
On the minified side, see #161
